assets/src/application/ui/input/InlineEditInput/InlineEditText.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { TextProps } from 'antd/es/typography/Text';
+
+import Editable from './Editable';
+
+const InlineEditText: React.FC<TextProps> = ({ ...rest }) => {
+	return <Editable inputType='text' ellipsis {...rest} />;
```

Comment:
passing `rest` this way isn't possible to override input type?

Reply:
That's restricted by types.